climada_petals/hazard/landslide.py

```diff
@@ -220,37 +233,49 @@ def set_ls_prob(self, bbox, path_sourcefile, corr_fact=10e6, n_years=500,
 
         Returns
         -------
-        self : climada.hazard.Landslide instance
+        Landslide : climada.hazard.Landslide instance
             probabilistic LS hazard
 
         See also
         --------
         sample_events_from_probs()
         """
-
+        
+        haz = Landslide()
```

Comment:
Why do you initiate the object with `Landslide()` instead of `cls()`?
```suggestion
        haz = cls()
```